app/helpers/tags_helper.rb

```diff
@@ -1,11 +1,12 @@
 module TagsHelper
+
   def tag_description(username)
-    "A collection created by @#{@username}"
+    "A collection created by @#{username}"
```

Comment:
this isn't breaking, but if username doesn't exist the twitter card will say "A collection created by "